agent/cache-types/streaming_health_services.go

```diff
@@ -28,6 +28,19 @@ type StreamingHealthServices struct {
 	deps MaterializerDeps
 }
 
+// RegisterOptions returns options with a much shorter LastGetTTL than the default.
+// Unlike other cache-types, StreamingHealthServices runs a materialized view in
+// the background which will receive streamed events from a server. If the cache
+// is not being used, that stream uses memory on the server and network transfer
+// between the client and the server.
+// The materialize view and the stream are stopped when the cache entry expires,
+// so using a shorter TTL ensures the cache entry expires sooner.
+func (c *StreamingHealthServices) RegisterOptions() cache.RegisterOptions {
+	opts := c.RegisterOptionsBlockingRefresh.RegisterOptions()
+	opts.LastGetTTL = 10 * time.Minute
```

Comment:
Calling this out since it is a pretty significant change. @kyhavlov noticed that since the default TTL is 3 days we would potentially be streaming a lot of data for no reason if we keep the materialized view around for that long.

Using a 10 minute TTL means that we might spend some extra time sending snapshots for semi-infrequent requests.

Reply:
The shorter TTL is different but with the blocking query implementation today we have the same problem of continuing the blocking query for 3 days and sending tons of data unnecessarily.

However the blocking query part was recently optimized to be able to return a not modified response to reduce the bandwidth needed. So for services queried every 11 minutes that do not change the blocking query approach should consume less bandwidth than the streaming approach due to being cached longer.

To me 10m seems like as sane of a default as 3d its just optimizing for lower memory utilization instead of more being in the cache. The problem is that we cannot know what the behavior of the applications utilizing our APIs will be with regards to how often they query and how long they expect the cache to be kept hot.

My thinking is that we should expose some tunables in the config for this with sane defaults that wont eat memory for days but allowing users who do care to override those defaults.

Reply:
I see you were already adding configuration of the cache so maybe a `refresh_timeout` or something might be nice here to allow configuring the default refresh timeout of all cache types. Although that could (and maybe should) be a different PR.

Reply:
I think that's a good idea. I'd like to do that in a follow up PR so that we can take more time to think about how we name the fields.